RegionalOverviews/overviews_reports_RDBES/fisheries_overviews_RDBES/annual_overview_RDBES_template.Rmd

```diff
@@ -410,33 +425,68 @@ sub_cl_2_top <- sub_cl_2_top %>%
 barplot(data = sub_cl_2_top, 
         x = "CLlandingLocation",
         y = "CLscientificWeight_1000ton_total",
-        title = "Landings (1000 t) by Harbour and Flag Country",
+    #    title = "Landings (1000 t) by Harbour and Flag Country",
         group = "CLvesselFlagCountry",
-        ylab = "Proportion of Landings (1000 t)",
+        ylab = "Proportion of Landings (%)",
         asPct = T)
 ```
 
-#### map - harbour
-#### map - harbour and flag country
+#### Map - harbour
+```{r, echo = FALSE, warning = FALSE, message = FALSE, results = "asis", dpi=100, fig.width=6,fig.height=6, fig.cap='**Figure 15**: Map with the landing weight by main harbours (top 20).'}
+
+sub_cl_top %>%  mutate(
+         analysis_type = 'sum',
+         type_of_threshold = 'none',
+         value_of_threshold = NA
+       )-> df
+
+
+result <- pointsMap_func2(df,
+                          title = '', #Landings (1000 t) by Harbour
+                          var_name = 'CLscientificWeight_1000ton',
+                          facet_name = 'CLyear',  # Poprawiono
+                          var_spatial_name = 'CLlandingLocation',
+                          spatial_dataset_name = 'Harbours',
+                          spatial_dataset_var_name = 'Harbour',
+                          plot_labels = TRUE,
+                          saveResults = FALSE,
+                          outputPath,
+                          Catch_group_name = NA,
+                          extraShp_name = 'StatRectshp',
+                          newVarName = NA,
+                          addToTitle = NA,
+                          RCGregion = 'BA'
+                         )
+
+  
+result
+
+```
+#### Map - harbour and flag country
+
+To be prepared for the next year.
 
 ### Landings abroad
-```{r riverplot aborad, echo = FALSE, warning = FALSE, message = FALSE,results = "asis", dpi=100, fig.width=9,fig.height=9}
-paste("River plot of CLscientificWeight_ton between CLvesselFlagCountry (left) and Harbour (right). Width of line is proportional to landings", sep="")  
+The plot below shows the flow of the landings between the vessel Flag Country (left), Harbour Country (center) and the Landing Country (right).
+
+```{r riverplot aboard, echo = FALSE, warning = FALSE, message = FALSE,results = "asis", dpi=100, fig.width=9,fig.height=9, fig.cap = '**Figure 16**: River plot of total landings between vessel Flag Country (left), Harbour Country (center) and landing Country (right). Width of each line is proportional to landings proportion. Harbour Country is the country code obtained from the CLlandingLocation.'}
```

Comment:
Width of each line is **proportional** to landings **proportion** -> sorry my mistake
I think it would be "Width of each line is proportional to landings"